duckduckgo/Android
# Hotfix/5.77.2

Task/Issue URL: https://app.asana.com/0/414730916066338/1199960174703530/f
Tech Design URL: 
CC: 

**Description**:
Fix for crash when `SystemLocationPermissionDialog` shown with our ddg icon vector drawable on devices with api<23

**Steps to test this PR**:
Using device api < 23
1. Visit our SERP
2. Search for "restaurants near me"
1. Ensure when dialog shown with ddg icon app does not crash


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
